content/docs/setup/kubernetes/download/index.md

```diff
@@ -51,3 +51,50 @@ To use the Istio release Helm chart repository, add the Istio release repository
 {{< text bash >}}
 $ helm repo add istio.io https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/charts/
 {{< /text >}}
+
+## istioctl Auto-Completion
```

Comment:
istioctl should be in backticks
Auto-Completion should be auto-completion (we don't capitalize words in headings)